cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -403,20 +402,20 @@ export default class SetPreferenceModal extends Component {
                       className="btn btn-primary float-xs-left"
                       disabled="disabled"
                     >
-                      <span className="fa fa-spinner fa-spin"></span>
+                      <span className="fa fa-spinner fa-spin" />
```

Comment:
Yikes, sorry about that. That's what I get for poking around pull requests that are none of my business. 😐